tests/sai_qualify/conftest.py

```diff
@@ -50,23 +50,27 @@
 RPC_CHECK_INTERVAL_IN_SEC = 4
 
 
+
 def pytest_addoption(parser):
     # sai test options
     parser.addoption("--sai_test_dir", action="store", default=None, type=str, help="SAI repo folder where the tests will be run.")
     parser.addoption("--sai_test_report_dir", action="store", default=None, type=str, help="SAI test report directory on mgmt node.")
     parser.addoption("--sai_test_container", action="store", default=None, type=str, help="SAI test container, saiserver or syncd.")
+    parser.addoption("--sai_test_keep_test_env", action="store_true", default=False, help="SAI test debug options. If keep the test environment in DUT and PTF.")
```

Comment:
```suggestion
    parser.addoption("--sai_test_keep_test_env", action="store_false", default=False, help="SAI test debug options. If keep the test environment in DUT and PTF.")
```
Please change to ```store_false``` and the ```default=False``` can be omitted.

Reply:
Ignore
```
'store_true' and 'store_false' - These are special cases of 'store_const' used for storing the values True and False respectively. In addition, they create default values of False and True respectively.
```